### ISSUE TITLE
skillCheck() is too procedural, move formatting to a function on the skill object

### ISSUE THREAD

**Issue by jlhall** [jlhall/D3Rand_Builder#29]
Switch statements can be an anti-pattern, and are annoying to read.

Swapping to a bunch of if else if etc. is just as bad...

Consider how skillCheck() is currently implemented, and in the context of issue #25, consider how an (eventual) change in tooltip generation could be paired with an enhancement to the new skill objects to remove a significant amount of code (formatting-functions, skill-check, etc.)

**Comment by jlhall** [comment]
In the current context of my other work, this is low priority, and I'd rather get skillCheck() and passiveCheck() covered with tests and then sit on these ideas for awhile once #28 is complete
